File: Data Structures & Algorithms/two-integer-sum/submission-1.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:

        seen = {}
        for x in range(0, len(nums)):
            if nums[x] not in seen:
                seen[nums[x]] = x
            need = target - nums[x]
            print(need)
            if need in seen and seen[need] != x:
                return [seen.get(need), x]

        

File: Data Structures & Algorithms/majority-element/submission-1.java
class Solution {
    public int majorityElement(int[] nums) {

        if (nums.length == 1){
            return nums[0];
        }

        int majPoint = nums.length / 2;

        Arrays.sort(nums);
        int k = 0;
        Set<Integer> seen = new HashSet<>();

        for (int i=1; i< nums.length; i++){
            int num = nums[i];
            if(!seen.contains(num)){
                seen.add(num);
                k = 0;
            }

            if (nums[i] == nums[i-1]){
                k++;
                if (k == majPoint){
                    return num;
                }
            }
        }
        return 0;


        
    }
}